### PR TITLE
Feat/connection list

### PR DESCRIPTION
**Pull request by dexturr**
## Feature connection list

Related to https://github.com/vegaprotocol/vegawallet-browser/issues/131

### CODE REVIEW

**Comment by dalebennett1992** [comment]
@dexturr could you mind adding unit tests to cover these ac? I think these should be in the unit tests

- I can see an explanation of what it means i.e. these dapps have permission to access my keys and connect to my wallet (<a name="1101-BWAL-077" href="#1101-BWAL-077">1101-BWAL-077</a>)
- I can see instructions how to connect to a Vega dapp (<a name="1101-BWAL-078" href="#1101-BWAL-078">1101-BWAL-078</a>)
- There is a way to see the dapps I could connect with (e.g. a link to https://vega.xyz/use ) (<a name="1101-BWAL-079" href="#1101-BWAL-079">1101-BWAL-079</a>)
- When I have no connections I can see that and still see instructions on how to connect to a Vega dapp (<a name="1101-BWAL-080" href="#1101-BWAL-080">1101-BWAL-080</a>)

**Comment by dalebennett1992** [comment]
image
Couple comments..
1. The question mark should be on the same line as the sentence it relates to of course, assuming this is a rogue space or something causing it to be pushed onto the second line.
2. There is no option to remove the connection there to my eyes
3. for urls with long names we should probably be restricting the width on the text, I do not think we should be allowing a sideways scroller

File: frontend/lib/modal-store.ts [hunk 1]
```diff
@@ -1,19 +1,35 @@
 import { create } from 'zustand'
 
+export interface ConnectionMessage {
+  origin: string
+  receivedAt: string
+}
+
+// TODO type this
```
Comment (edd) [inline frontend/lib/modal-store.ts:8]: **Thought (non-blocking)**: yes

**Review by edd — APPROVED** [review]
No useful feedback. Looks good, nice tests

**Review by dalebennett1992 — APPROVED** [review]

**Pull request merged**